docs/pages/enroll-resources/application-access/okta/app-and-group-sync.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Okta apps and groups sync
+description: Details on Okta integration apps and groups synchronization as Teleport Access Lists
+---
+
+Okta has its own permissions system that doesn't directly map into Teleport. This
+includes permissions granted by Okta groups along with the assignment of users to individual
+applications within Okta. To model this within Teleport, an administrator would typically need
+to carefully craft a labeling system to attach to various Okta apps and groups and a set of
+roles to go along with them.
+
+With the synchronization of Okta apps and groups ass Teleport Access List feature in the Okta
+integration, this work is performed for you. We'll discuss the details of how this works.
+
+## Enrollment
+
+To enroll the Okta apps and groups sync follow the instructions of *Level 1. SSO*, *Level 2. SCIM*
+(optional), *Level 3. User sync* and *Level 4. App and Group sync* enrollment in the [Okta
+integration setup guide](./integration-enrollment.mdx).
+
+## Synchronizing Access Lists from Okta groups and applications
```

Comment:
If this is an architecture guide, we should move it to `reference/architecture`. An alternative would be to move the general architecture information to the section index page at `enroll-resources/application-access/okta/okta.mdx`. This would provide a conceptual overview of the feature so a user has a mental model to work with when navigating through the different pages.